docs/source/conf.py

```diff
@@ -33,6 +34,7 @@
     "sphinx_scylladb_theme",
     "sphinx_multiversion",  # optional
     "myst_parser",  # for converting .md to .rst,
+    "rst_jinja"
```

Comment:
What does rst_jinja do? Is it how you use the template links?

Reply:
It enables use to do templating in the rst files and specifically adds int the env var that we need from the multi version extension